src/scripts/helpers/swipe-tables.js

```diff
@@ -0,0 +1,50 @@
+(function () {
+  'use strict';
+
+  var tableLeftPos, startSwipingPos, swipeRange;
+    
+  function handleSwipe (target) {
+    var mouseX = window.event.clientX;
+    var swipeDistance = startSwipingPos - mouseX;
+    if (swipeDistance <= swipeRange && swipeDistance >= (-1 * swipeRange)) {
+      target.scrollLeft += swipeDistance;
+    }
+  }
+  
+  function startSwiping (target) {
+    var offsets = target.getBoundingClientRect();
+    tableLeftPos = offsets.left;
+    startSwipingPos = window.event.clientX;
+    swipeRange = target.getElementsByTagName('table')[0].offsetWidth - target.offsetWidth;
+    target.onmousemove = function() {
+      handleSwipe(target);
+    };
+  }
+  
+  function stopSwiping (target) {
+    target.onmousemove = null;
+  }
+
+  if( !/Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent) ) {
+    var tablesToSwipe = Array.prototype.filter.call(document.getElementsByClassName('os-table'), function(el){
```

Comment:
Could this be written more succinctly using `[...document.querySelectorAll('.os-table')].filter(el => ...)` syntax? (or coffeescript, but might not be worth learning)

See main review comment for alternate suggestions that may not need this code

Reply:
`Uncaught TypeError: document.querySelectorAll(...).filter is not a function`
```
Array.isArray(document.querySelectorAll('.os-table'))
false
```
I'ts not an Array so I'am not sure if there is another work around for this than calling `Array.prototype....call`

Reply:
`[...xyz]` converts an Iterable (`xyz`) into an array using the [Array Spread](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) syntax. I think `[...document.querySelectorAll('.os-table')].filter(el => ...)` should work

Reply:
This would work but travis... :D 
```
29 |    var tablesToSwipe = [...document.getElementsByClassName('os-table')].filter(el => {
                                  ^ [W119] 'spread/rest operator' is only available in ES6 (use 'esversion: 6').
     29 |    var tablesToSwipe = [...document.getElementsByClassName('os-table')].filter(el => {
                                                                                            ^ [W119] 'arrow function syntax (=>)' is only available in ES6 (use 'esversion: 6').
```